### PR TITLE
refactor(dms/kafka_topic): fix lint issues

### PR DESCRIPTION
**Pull request by chengxiangdong**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

Fix some lint issues in `huaweicloud_dms_kafka_topic`.

**Release note**:

```release-note
NONE
```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dms' TESTARGS='-run TestAccDmsKafkaTopic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms -v -run TestAccDmsKafkaTopic -timeout 360m -parallel 4
=== RUN   TestAccDmsKafkaTopic_basic
=== PAUSE TestAccDmsKafkaTopic_basic
=== CONT  TestAccDmsKafkaTopic_basic
--- PASS: TestAccDmsKafkaTopic_basic (635.38s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       635.495s
```

### CODE REVIEW

**Comment by chengxiangdong** [comment]
All work done and ready for review.

File: huaweicloud/services/dms/resource_huaweicloud_dms_kafka_topic.go [hunk 1]
```diff
@@ -221,19 +223,19 @@ func resourceDmsKafkaTopicDelete(ctx context.Context, d *schema.ResourceData, me
 
 // resourceDmsKafkaTopicImport query the rules from HuaweiCloud and imports them to Terraform.
 // It is a common function in waf and is also called by other rule resources.
-func resourceDmsKafkaTopicImport(_ context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData,
+func resourceDmsKafkaTopicImport(_ context.Context, d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData,
 	error) {
 	parts := strings.SplitN(d.Id(), "/", 2)
 	if len(parts) != 2 {
-		err := fmtp.Errorf("Invalid format specified for DMS kafka topic. Format must be <instance id>/<topic name>")
+		err := fmt.Errorf("Invalid format specified for DMS kafka topic. Format must be <instance id>/<topic name>")
```
Comment (hmbig2) [inline huaweicloud/services/dms/resource_huaweicloud_dms_kafka_topic.go]: lowercase the first letter

**Pull request merged**